code/modules/projectiles/guns/pistols.dm

```diff
@@ -148,6 +148,22 @@
 	scatter_unwielded = SCATTER_AMOUNT_TIER_6
 	damage_mult = BASE_BULLET_DAMAGE_MULT
 
+/obj/item/weapon/gun/pistol/b92fs/custom
```

Comment:
No need for a special gun to the CMP, instead look at replacing the normal colony one if the new sprite is good and fitting for the tone/setting.